modules/rosa-hcp-vpc-terraform.adoc

```diff
@@ -58,10 +58,8 @@ $ terraform apply rosa.tfplan
 ----
 $ export SUBNET_IDS=$(terraform output -raw cluster-subnets-string)
 ----
-
-.Verification
-
-* You can verify that the variables were correctly set with the following command:
++
+Verify that the variables were correctly set with the following command:
```

Comment:
It might make more sense to have this as bullets because there are multiple commands/steps in this optional procedure. You did it as bullets above with "Account roles". 
![image](https://github.com/openshift/openshift-docs/assets/119696027/8e99e8ef-0078-4fc5-aa4b-b0ffcbf6cde3)